gnd/src/test/java/com/google/android/gnd/persistence/local/DateTimeResponseTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.persistence.local;
+
+import static com.google.android.gnd.model.observation.DateResponse.DATE_FORMAT;
+import static com.google.android.gnd.model.observation.TimeResponse.TIME_FORMAT;
+
+import com.google.android.gnd.rx.SchedulersModule;
+import com.google.common.truth.Truth;
+import dagger.hilt.android.testing.HiltAndroidTest;
+import dagger.hilt.android.testing.HiltTestApplication;
+import dagger.hilt.android.testing.UninstallModules;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import java.util.Locale;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+@HiltAndroidTest
+@UninstallModules({SchedulersModule.class, LocalDatabaseModule.class})
+@Config(application = HiltTestApplication.class)
+@RunWith(RobolectricTestRunner.class)
+public class DateTimeResponseTest {
```

Comment:
It appears these tests aren't testing any behaviors of the application. Instead, it appears they're testing the system date/time format and parse methods. Was this intentional?

Reply:
Over here the format is checked of Date and time. TIME_FORMAT.format(originalDate) is from TimeResponse. DATE_FORMAT.format(originalDate) is from DateResponse.

Reply:
over here i am just checking the format is matching the time and date format which is defined in DateResponse and TimeResponse.

Reply:
only Internal formatter instance are used to check the formatting rest public methods are used to validate that.